tokio-rustls/src/server.rs

```diff
@@ -136,3 +138,13 @@ where
         self.get_ref().0.as_raw_fd()
     }
 }
+
+#[cfg(windows)]
+impl<IO> AsRawHandle for TlsStream<IO>
```

Comment:
I think it should be `AsRawSocket`? like [TcpStream](https://doc.rust-lang.org/stable/std/net/struct.TcpStream.html#impl-AsRawSocket)